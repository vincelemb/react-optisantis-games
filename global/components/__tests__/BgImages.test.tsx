import React from 'react';
import { shallow } from 'enzyme';
import { BgImage } from '..';

describe('<BgImage />', () => {
    const props = { imageUrl: 'hero.jpg' };
    const wrapper = shallow(<BgImage {...props} />);
    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });
    it('Test id of button ', () => {
        let parameter = props.imageUrl;
        let extension = ['.jpg', '.jpeg'];
        expect(parameter.endsWith(extension[0]) || parameter.endsWith(extension[1])).toBe(true);
    });
});
