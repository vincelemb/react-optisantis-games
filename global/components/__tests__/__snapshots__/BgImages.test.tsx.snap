// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BgImage /> should render 1`] = `
<div
  className="_bg-cover _h-screen _overflow-auto"
  style={
    Object {
      "backgroundImage": "url(hero.jpg)",
    }
  }
/>
`;
